.github/workflows/ci.yml

```diff
@@ -27,8 +27,14 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: Install deps and run tests
-        run: npm cit
+      - name: Install deps
+        run: npm ci
+
+      - name: Run tests
+        run: npm test
+
+      - name: Run benchmarks
+        run: npm run bench
```

Comment:
It would be rad if there were a github action to dump the contents of these benchmarks into a PR comment.

Reply:
Yeahh 100%. Like i said in slack, i think you'd have to store the benchmark results somewhere and compare the current run to a known baseline or something.